stripper/ze_sky_athletic_adv_v9_12.cfg

```diff
@@ -38,3 +38,17 @@ modify:
 		"OnNotEqualTo" "tutorial_brushDisable30-1"
 	}
 }
+
+;This map has already been ported to css, why is it here?
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "10021"
+	}
+	delete:
+	{
+		"OnMapSpawn" "server commandCommandsay ** NOT ALLOWING PORTING TO CSS ***10-1"
+	}
+}
```

Comment:
This message is stating the mappers wish to not have the map ported to CS:S, just because somebody violated the mappers wishes and ported it anyways doesn't mean it should be removed.